libkbfs/conflict_resolver.go

```diff
@@ -195,8 +206,16 @@ func (cr *ConflictResolver) Resolve(unmerged MetadataRevision,
 		return
 	}
 
+	cr.inputLock.Lock()
+	defer cr.inputLock.Unlock()
```

Comment:
Ah I forgot about the channel send blocking, that is a deadlock, will fix.  Good catch.

> Does that sound right?

That shouldn't result in a bug, because at the start of each PR it pulls the full set of unmerged commits (i.e., what it resolves doesn't actually depend on the input, the input is just there to help decide what resolves it can reject completely).  Does that make sense?